src/azul/service/manifest_service.py

```diff
@@ -1203,8 +1223,7 @@ def _write(file: JSON, is_related_file: bool = False):
                 '--write-out "Downloading to: %{filename_effective}\\n\\n"'
             ]
             if self.authentication is not None:
-                header = self.authentication.as_http_header()
-                curl_options.append(f'--header {self._option(header)}')
+                curl_options.append(' '.join(self._authentication_option(self.authentication)))
```

Comment:
I don't think the token should be embedded in the manifest.

Reply:
I agree but I think it would be better to cover this in a separate PR